microceph/client/wrap.go

```diff
@@ -55,5 +56,10 @@ func (c ClientImpl) DeleteClusterMember(cli *microCli.Client, name string, force
 	return cli.DeleteClusterMember(context.Background(), name, force)
 }
 
+// PutOsds wraps the PutOsds function
+func (c ClientImpl) PutOsds(cli *microCli.Client, up bool, target string) error {
```

Comment:
I have some doubts,
1. Where is this method being used ?
2. Shouldn't the client functions for maintenance API be also a part of this interface ?

Reply:
oh, I guess you are right! These (api + client wrapper + mock interface for osds) WERE used before, and now they are not used. I will remove them

Reply:
Removed

Reply:
Thanks